tests/__fixtures__/php-tags/php-wrapper-with-extra-closing.php

```diff
@@ -0,0 +1,5 @@
+<?php echo "Some to be ?>happy about"; ?>
+----
+@php
+ echo "Some to be ?>happy about";
+@endphp
```

Comment:
Please add a newline at the end of the file
```suggestion
@endphp

```